status-whisper-mailserver-spec.md

```diff
@@ -43,35 +43,35 @@ In the Whisper envelope's payload section, there MUST be RLP-encoded information
 [ Lower, Upper, Bloom, Limit, Cursor ]
 ```
 
-`Lower`: 4-byte wide unsigned integer (UNIX time in seconds; oldest requested envelope's creation time)
-`Upper`: 4-byte wide unsigned integer (UNIX time in seconds; newest requested envelope's creation time)
-`Bloom`: 64-byte wide array of Whisper topics encoded in a bloom filter to filter envelopes
-`Limit`: 4-byte wide unsigned integer limiting the number of returned envelopes
+`Lower`: 4-byte wide unsigned integer (UNIX time in seconds; oldest requested envelope's creation time)  
```

Comment:
Trailing whitespaces.

Reply:
Intended, markdown newline (either that or change in a table as is done in the Waku version)